go/interpreter/lfvm/instructions.go

```diff
@@ -22,16 +22,15 @@ func opStop() status {
 	return statusStopped
 }
 
-func opRevert(c *context) status {
-	c.resultOffset = *c.stack.pop()
-	c.resultSize = *c.stack.pop()
-	return statusReverted
-}
-
-func opReturn(c *context) status {
-	c.resultOffset = *c.stack.pop()
-	c.resultSize = *c.stack.pop()
-	return statusReturned
+func opReturningOp(c *context, status status) (status, error) {
```

Comment:
The status argument is only passed in to be returned. This could be dropped. The caller knows the result state already.

Reply:
right, this was to forward the right status, and error, but as you mention it can be avoided.